ldk/javascript/examples/self-test-loop/src/selfTest.ts

```diff
@@ -1,6 +1,6 @@
 import { keyboard } from '@oliveai/ldk';
 import { emitTestWhisper } from './testingFixtures/testWhisper';
-import WhisperCloser from './testingFixtures/WhisperCloser';
+import WhisperCloser from './testingFixtures/whisperCloser';
```

Comment:
Hummm, it seems to be some git glitch? Is there any difference here?

Reply:
case is different

Reply:
This bug doesn't show up for most people on their systems because MacOS HDs are generally case-insensitive (at least initally). Mine is case-sensitive so it flagged as an error on my system. We'd have seen this error earlier if the self test loop was built in CI.